packages/workflow/src/workflow.ts

```diff
@@ -948,6 +962,7 @@ export function makeContinueAsNewFunc<F extends Workflow>(
  *
  *```ts
  *import { continueAsNew } from '@temporalio/workflow';
+import { SearchAttributeValue } from '../../common/lib/interfaces';
```

Comment:
Why add this `import` to this specific sample? I'd remove.

Reply:
Not sure what happened there, probably some code I moved around that I forgot to remove.